Resources/config/ckeditor.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="cmf_content.form.extension.ckeditor"
+            class="Symfony\Cmf\Bundle\ContentBundle\Form\Extension\CKEditorExtension">
```

Comment:
CKEditorExtension accepts options in the constructor. i think we still have the options in the Configuration.php but should pass them to the extension here.